### DISCUSSION TITLE
Github pages tutorial clear, but where is the link address of this page?

### DISCUSSION THREAD

**Discussion by bbjoony** [skills/.github#51] [General]
Thanks to this course! I've tried making a Github blog several times before, but I couldn't do it. Due to this tutorial, I could finally see my first GitHub blog. But I can't find my blog address easily. If you guys add the external address which makes other guys(even including me) connect and see that blog, it would be very helpful. Thanks!

**Comment by ecodoc** [comment]
I totally agree. Where is the page?

**Comment by sinsukehlab** [comment]
From the root page of the repository, click **Environments** on the right to go to the deployments page.
Click the **View deployment** button, and you will get to the published page.

**Reply by sinsukehlab** [reply]
I recommend you to finish the Communicate using Markdown course.
There is a brief instruction at the end.

**Reply by sinsukehlab** [reply]
PR skills/github-pages#20 should update the instruction.

**Comment by 6-ril** [comment]
Hello, 
I finally found by myself, but it could be great to include this information in the tutorial.
Probably at the end.
If I'm correct, I may suggest the modification with a pull request !

**Reply by Pitxunet** [reply]
Good idea! It's not very intuitive for beginners. Otherwise I found the tutorial very useful to set up the page

**Reply by Starr79** [reply]
I enjoyed this
